### ISSUE TITLE
Server constructor error in browser

### ISSUE THREAD

**Issue by sreuland** [esteblock/soroban-react#30]
getting this error in browser when using the soroban client sdk during initialization:

```
Unhandled Runtime Error
TypeError: SorobanClient.Server is not a constructor

eval
node_modules/@soroban-react/core/dist/SorobanContext.js (22:0)
```

on new 4.4.1 versions of packages.

initial bug reported here - https://github.com/esteblock/soroban-react/pull/28#issuecomment-1554847236

**Comment by Shaptic** [comment]
Is this still true in 0.6.1? I can't seem to reproduce it.

**Comment by esteblock** [comment]
I will check

**Comment by Shaptic** [comment]
er, please use 0.7.0 instead when you check!

**Comment by esteblock** [comment]
Hello! I am checking with soroban-client 0.7.0 here: https://github.com/esteblock/soroban-example-dapp/tree/soroban-client-0.7.0

I am using

```
 "@soroban-react/chains": "4.5.1-alpha.0",
    "@soroban-react/contracts": "4.5.1-alpha.0",
    "@soroban-react/core": "4.5.1-alpha.0",
    "soroban-client": "0.7.0"
```

All of them use soroban-client@0.7.0

However I still have the same problem

```
Unhandled Runtime Error
TypeError: soroban_client_1.default.Server is not a constructor

Call Stack
eval
node_modules/@soroban-react/core/dist/SorobanContext.js (22:0)
```

![Screenshot from 2023-05-24 11-41-22](https://github.com/esteblock/soroban-react/assets/2274485/094bd73f-d2e4-4cc8-8911-5f1e84d456d0)

**Comment by esteblock** [comment]
I think it has to do with some webpack changes introduced in https://github.com/stellar/js-soroban-client/compare/v0.5.1...v0.6.0

**Comment by esteblock** [comment]
Please check @Shaptic @sreuland

**Comment by Shaptic** [comment]
Ugh I literally don't understand. I checked out the branch and added `console.log` statements to `node_modules/@soroban-react/core/dist/SorobanContext.js` and `soroban_client_1.default.Server` totally works when you `npm run build`, but something completely borks when you `npm run serve`.

**Comment by esteblock** [comment]
When I install soroban-client@0.5.1 works OK, both with create-react-app and next

https://github.com/esteblock/create-react-app-soroban-client/tree/soroban-client%400.5.1
https://github.com/esteblock/my-next-app-soroban-client/tree/soroban-client%400.5.1

**Comment by Shaptic** [comment]
This should be resolved by upgrading to v0.7.2 (v0.7.1 works, as well, but the other patch fix will do no harm)!
